doc/ext/viewcode.rst

```diff
@@ -62,3 +65,17 @@ There is an additional config value:
       Some reader's rendering result are corrupted and
       `epubcheck <https://github.com/IDPF/epubcheck>`_'s score
       becomes worse even if the reader supports.
+
+.. event:: viewcode-find-source (app, modname)
+
+   .. versionadded:: 1.8
+
+   Find the source code for a module.
+   The event returns a tuple of the source code itself,
```

Comment:
I feel "A event handler for this event should return" is better.

Reply:
Good point. I've made this change.